languages/elixir/exercises/concept/multiple-clause-functions/.docs/introduction.md

```diff
@@ -0,0 +1,58 @@
+Elixir allows programmers to write code that follows closely to the **Open-Close Principle**. That is, software should be open for extension, but closed for modification. This is an important principle, because as your software grows in size and complexity, changes to that software are more difficult to make.
```

Comment:
Mentioning the open/closed principle doesn't seem necessary to learn the concept in my opinion. If you think it's important, maybe put it in after.md instead?

Reply:
Sometimes I think it is important to learn why something is important before learning it to give motivation to understand the material.  I can move it to after.md